src/EA.Weee.Web/Areas/Scheme/ViewModels/TransferEvidenceNoteDataViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.ViewModels
+{
+    using EA.Weee.Core.Scheme;
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+
+    public class TransferEvidenceNoteDataViewModel
+    {
+        [Required(ErrorMessage = "Select a PCS")]
```

Comment:
Unit tests for properties on the model

Reply:
they will be added, just in a different branch